src/test/java/catan/services/impl/PlayServiceImplTest.java

```diff
@@ -206,7 +217,11 @@ public void shouldFailWhenBuildRoadNotNearOwnNeighbourCityOrRoad() throws GameEx
 
         try {
             // WHEN
-            playService.buildRoad(gameUser1.getUser(), "1", "7");
+
+            HashMap<String, String> params = new HashMap<String, String>();
+            params.put("edgeId", "7");
+
+            playService.processAction(GameUserActionCode.BUILD_ROAD, gameUser1.getUser(), "1", params);
             fail("PlayException with error code '" + PlayServiceImpl.ERROR_CODE_ERROR + "' should be thrown");
         } catch (PlayException e) {
             // THEN
```

Comment:
that was a result of merging. merge was first before code review, so i decided you just forgot to fix String param to HashMap